s3backgrounddelete/s3backgrounddelete/cortx_s3_config.py

```diff
@@ -95,106 +96,188 @@ def cache_credentials(self):
 
     def get_config_version(self):
         """Return version of S3 background delete config file or KeyError."""
-        if 'version_config' in self._config and self._config['version_config']['version']:
-            return self._config['version_config']['version']
-        else:
+        #if 'version_config' in self._config and self._config['version_config']['version']:
+        #    return self._config['version_config']['version']
+        #else:
+        #    raise KeyError("Could not parse version from config file " + self._conf_file)
+        try:
+          if self.get_config('version_config>version'):
+            return self.get_config('version_config>version')
+        except Exception as e:
             raise KeyError("Could not parse version from config file " + self._conf_file)
 
     def get_logger_directory(self):
         """Return logger directory path for background delete from config file or KeyError."""
-        if 'logconfig' in self._config and self._config['logconfig']['logger_directory']:
-            return self._config['logconfig']['logger_directory']
-        else:
+        #if 'logconfig' in self._config and self._config['logconfig']['logger_directory']:
+        #    return self._config['logconfig']['logger_directory']
+        #else:
+        #    raise KeyError(
+        #        "Could not parse logger directory path from config file " +
+        #        self._conf_file)
+        try:
```

Comment:
if and try do not go together - use only one

Reply:
done